packages/@expressive-code/plugin-frames/src/utils.ts

```diff
@@ -25,7 +25,7 @@ export function frameTypeFromString(input: string) {
 
 export const LanguageGroups = {
 	code: ['astro', 'cjs', 'htm', 'html', 'js', 'jsx', 'mjs', 'svelte', 'ts', 'tsx', 'typescript', 'vb', 'vue', 'vue-html'],
-	terminal: ['ansi', 'bash', 'bat', 'batch', 'cmd', 'console', 'powershell', 'ps', 'ps1', 'psd1', 'psm1', 'sh', 'shell', 'shellscript', 'shellsession', 'zsh'],
+	terminal: ['ansi', 'bash', 'bat', 'batch', 'cmd', 'console', 'nu', 'powershell', 'ps', 'ps1', 'psd1', 'psm1', 'sh', 'shell', 'shellscript', 'shellsession', 'zsh'],
```

Comment:
According to the [Shiki docs](https://shiki.style/languages), `nu` is an alias of `nushell`, so it would be good to add the long form here as well.